src/@next/Banner/Banner.stories.tsx

```diff
@@ -43,3 +43,90 @@ WithButtons.args = {
   title: 'Title',
   buttonLoading: false,
 };
+
+const FixedBannerTemplate: Story<BannerProps> = args => {
+  const action1 = <Button>Action 1</Button>;
+  const action2 = <Button>Action 2</Button>;
+
+  return (
+    <>
+      <Banner
+        title="Title"
+        type="fixed"
+        action={action1}
+        secondaryAction={action2}
+        status="success"
+        {...args}
+      >
+        {defaultText}
+      </Banner>
+      Etiam vel mauris vehicula, bibendum elit id, rutrum sapien. Fusce
+      efficitur dolor vel lorem gravida lacinia. Donec at malesuada diam. Nullam
+      fermentum arcu at ligula ultricies sagittis. Sed erat dui, tempus in
+      luctus ac, venenatis et enim. Cras sed sollicitudin sem. Etiam id quam
+      vitae nisl lacinia sollicitudin vitae suscipit massa. Nullam quis viverra
+      nisl. Integer tempus hendrerit elementum. Maecenas pharetra risus metus.
+      Interdum et malesuada fames ac ante ipsum primis in faucibus. Cras dictum
```

Comment:
I noticed that we have several lorem ipsum texts like this used for stories in various other components. Rather than writing something like this and make the code a bit dirty, how about if we make utility function to generate lorem ipsum text (e.g., just need to call `generateLoremIpsum(300)`, where 300 is the number of words).

A downside is it might take a while (O(n)) where n is the number of words to do this. (e.g., store lorem as a list, slice the list and join it together as strings with space).